modules/@ergonode/workflow/src/components/Buttons/UpdateProductStatusTranslationButton.vue

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright © Ergonode Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <FeedbackProvider
+        :errors="errors"
+        :change-values="changeValues">
+        <template #default="{ hasValueToSave }">
+            <Button
+                data-cy="submit"
+                :title="$t('@Workflow._.submit')"
+                :floating="{ bottom: '24px', right: '24px' }"
+                :disabled="!isAllowedToUpdate"
+                @click.native="onSubmit">
+                <template #prepend="{ color }">
+                    <IconSpinner
```

Comment:
We are trying to avoid dynamic components whenever it's possible -> we are only using them for dynamically imported / generated components from modules